### PR TITLE
JAMES-2334 Demonstrate that routing is working

### PR DESCRIPTION
**Pull request by aduprat**
Only last commit.

### CODE REVIEW

File: server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RoutingTest.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            channel1.queueBind(queue1, EXCHANGE_NAME, CONVERSATION_1);
+            channel1.queueBind(queue1, EXCHANGE_NAME, CONVERSATION_2);
+
+            String queue2 = channel2.queueDeclare().getQueue();
+            // 2 will follow discussion 3 and 2
+            channel2.queueBind(queue2, EXCHANGE_NAME, CONVERSATION_3);
+            channel2.queueBind(queue2, EXCHANGE_NAME, CONVERSATION_2);
+
+            String queue3 = channel3.queueDeclare().getQueue();
+            // 1 will follow discussion 3 and 4
```
Comment (chibenwa) [inline server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RoutingTest.java]:
```
// 3 will follow...
```

File: server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RoutingTest.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            channel3.queueBind(queue3, EXCHANGE_NAME, CONVERSATION_3);
+            channel3.queueBind(queue3, EXCHANGE_NAME, CONVERSATION_4);
+
+            String queue4 = channel4.queueDeclare().getQueue();
+            // 1 will follow discussion 1 and 2
```
Comment (chibenwa) [inline server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RoutingTest.java]:
```
// 4 will follow discussion 1 and 4
```

File: server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RoutingTest.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            channel4.queueBind(queue4, EXCHANGE_NAME, CONVERSATION_1);
+            channel4.queueBind(queue4, EXCHANGE_NAME, CONVERSATION_4);
+
+            // 1 will produce 100 messages
```
Comment (chibenwa) [inline server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RoutingTest.java]: Inacurate comment. It's more:

```
// Publish a message on each discussion
```

**Comment by mbaechler** [comment]
merged in another PR

**Pull request closed without merging**